tests/unit/drm_plugin_test.sh

```diff
@@ -284,15 +287,25 @@ function test_get_available_connectors_remote()
   output=$(
     function cmd_remotely()
     {
-      printf '/sys/class/drm/card0-%s-1\n' 'DP'
-      printf '/sys/class/drm/card0-%s-1\n' 'eDP'
+      cmd_run=$(printf "%s" "$1" | cut -d ' ' -f1)
```

Comment:
Why not use static results? Personally, I don't like the idea of inserting complex code into a mock function like this. Consider reworking this test to make it more straightforward while you address these shellchecks.